starter-code/src/App.js

```diff
@@ -1,21 +1,35 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import 'bulma/css/bulma.css'
+import foods from './foods.json'
 import './App.css';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
 
 class App extends Component {
+  state = {
+    appFoods: foods
+  }
+
+  //APP NECESITA FUNCION PARA CAPTAR NEW FOOD
+  addNewFood = (food) => {
+    let addList = [...this.state.appFoods]
+    addList.push(food)
```

Comment:
estas dos lineas son lo mismo que:
`const addList = [...this.state.appFoods, food]`